tests/backup/backup_helper.go

```diff
@@ -2026,3 +2028,28 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// LicenseCountCheck verifies the license count for backup
+func LicenseCountCheck(ctx context.Context, licenseInspectRequestObject *api.LicenseInspectRequest, expectedLicenseCount int64) error {
```

Comment:
Instead of taking `licenseInspectRequestObject` as the argument, you can take `orgID` as the argument and then construct as follows:
```go
licenseInspectRequestObject = &api.LicenseInspectRequest{
			OrgId: orgID,
}
```

Reply:
done